common/pom.xml

```diff
@@ -16,6 +16,8 @@
     <url>https://github.com/lightstep/lightstep-tracer-java-common</url>
 
     <properties>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
Naive question: will this cause an issue with java 1.7 compatibility?

I checked back to https://lightstep.slack.com/archives/CTKS22ZE0/p1583389860052700?thread_ts=1583259860.021800 and it sounds like we can keep 1.7 compatibility?

Reply:
The Oshi library we're using to collect metrics supports jdk1.8+. This means the module has to be built with jdk1.8+ compatibility. The entrypoint into the metrics engine checks for the jdk version [here](https://github.com/lightstep/lightstep-tracer-java-common/pull/123/files#diff-870d4e5fe863929ff5a953f3a0057433R24-R27). This means that environments running jdk1.7 will not support metrics.

Reply:
I think it's ok - for now at least, as we don't really support Java 1.7 at the moment at all. This has been a recurring conversation but we haven't decided anything on that front yet.

Reply:
sounds good, thanks!